scripts/exregional_run_fcst.sh

```diff
@@ -519,7 +639,12 @@ if [ ${WRITE_DOPOST} = "TRUE" ]; then
     post_fn_suffix="GrbF${fhr_d}"
     post_renamed_fn_suffix="f${fhr}${post_mn_or_null}.${POST_OUTPUT_DOMAIN_NAME}.grib2"
 
-    fids=( "prslev" "natlev" )
+    if [ "${CPL_AQM}" = "TRUE" ]; then
+      fids=( "cmaq" )
+    else
+      fids=( "prslev" "natlev" )
+    fi
+
```

Comment:
RRFS_dev also uses a different convention for naming post files (`bgdawp`, `bgrd3d`) but those can be renamed to (`prslev`, `natlev`) etc.